wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/EventLogScreen.kt

```diff
@@ -0,0 +1,99 @@
+package com.android.identity_credential.wallet.ui.destination.document
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.runtime.remember
+import com.android.identity_credential.wallet.DocumentModel
+import com.android.identity_credential.wallet.EventInfo
+import com.android.identity_credential.wallet.R
+import com.android.identity_credential.wallet.navigation.WalletDestination
+import com.android.identity_credential.wallet.ui.KeyValuePairText
+import com.android.identity_credential.wallet.ui.ScreenWithAppBarAndBackButton
+
+private const val TAG = "EventLogScreen"
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun EventLogScreen(documentId: String, documentModel: DocumentModel, onNavigate: (String) -> Unit) {
+    val eventInfos = remember {
+        mutableStateListOf<EventInfo>().apply {
+            addAll(documentModel.getEventInfos(documentId))
+        }
+    }
+
+    Box(
+        modifier = Modifier.fillMaxHeight()
+    ) {
+        ScreenWithAppBarAndBackButton(
+            title = stringResource(R.string.document_info_screen_menu_activities_log),
+            onBackButtonClick = { onNavigate(WalletDestination.PopBackStack.route) }
+        ) {
+            if (eventInfos.isEmpty()) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.Center,
+                ) {
+                    Text(
+                        modifier = Modifier.padding(8.dp),
+                        text = stringResource(R.string.document_info_screen_menu_activities_log_screen_empty),
+                        style = MaterialTheme.typography.titleLarge,
+                        textAlign = TextAlign.Center
+                    )
+                }
+            }
+        }
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(top = 52.dp)  // Padding to avoid overlap with the AppBar
```

Comment:
This doesn't look right, I think you need just need to move your UI so it's a child of the above.

Reply:
Done.